native/src/widget/scrollable.rs

```diff
@@ -357,11 +393,20 @@ impl State {
     }
 
     /// Returns whether the scrollbar is currently grabbed or not.
-    pub fn is_scrollbar_grabbed(&self) -> bool {
-        self.scrollbar_grabbed_at.is_some()
+    pub fn currently_grabbed(&self) -> bool {
+        self.scroller_grabbed_at.is_some()
     }
 }
 
+#[derive(Debug, Clone, Copy)]
+/// What the mouse is grabbing on the scrollbar
+pub enum ScrollbarGrab {
+    /// The mouse is grabbing the background
+    Background,
+    /// The mouse is grabbing the scroller
+    Scroller,
+}
```

Comment:
I think this type describes a part of the scrollbar. It seems you can have one even when the user isn't grabbing anything.

I think naming it someting like `ScrollbarItem` and then implementing a `ScrollbarItem::from_cursor_position` (the current `Renderer::scrollbar_grab`) would keep it opaque, more intuitive, and safer to use.